.github/workflows/run-daily-target-aggre.yaml

```
@@ -0,0 +1,92 @@
+    - cron: "0 * * * *"
+
+jobs:
+  velog-posts-daily-stats-batch:
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.4)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)